hscontrol/policy/v2/types_test.go

```diff
@@ -765,6 +765,92 @@ func TestResolvePolicy(t *testing.T) {
 			toResolve: Wildcard,
 			want:      []netip.Prefix{tsaddr.AllIPv4(), tsaddr.AllIPv6()},
 		},
+		{
+			name:      "autogroup-member-basic",
+			toResolve: ptr.To(AutoGroup(AutoGroupMember)),
+			nodes: types.Nodes{
+				{
+					User: users["testuser"],
+					IPv4: ap("100.100.101.1"),
+				},
+				{
+					User:       users["testuser"],
+					ForcedTags: []string{"tag:test"},
+					IPv4:       ap("100.100.101.2"),
+				},
+				{
+					User: users["testuser"],
+					Hostinfo: &tailcfg.Hostinfo{
+						RequestTags: []string{"tag:test"},
+					},
+					IPv4: ap("100.100.101.3"),
+				},
```

Comment:
Per the comment above, this should be included, as this test would not populate the `tagOwner` map.

We might need a version of the ToResolve test that will take full packetfilter and resolve it.

Reply:
I reworked the test logic. This should test the functionality better. But honestly, I'm not really sure if this is enough or if a more robust testing should be added.